.yarnrc.yml

```diff
@@ -17,4 +17,4 @@ plugins:
   - path: .yarn/plugins/@yarnpkg/plugin-version.cjs
     spec: '@yarnpkg/plugin-version'
 
-yarnPath: .yarn/releases/yarn-2.1.1.cjs
+yarnPath: .yarn/releases/yarn-2.4.1.cjs
```

Comment:
For some random reason, I couldn't install Preconstruct on a previous version because installs were failing with:
```
➤ YN0001: │ Error: fsevents@patch:fsevents@npm%3A2.3.2#builtin<compat/fsevents>::version=2.3.2&hash=495457: Cannot apply hunk #1
    at d (/radix-ui-primitives/.yarn/releases/yarn-2.1.1.cjs:2:190674)
    at async /radix-ui-primitives/.yarn/releases/yarn-2.1.1.cjs:2:189912
    at async h (/radix-ui-primitives/.yarn/releases/yarn-2.1.1.cjs:2:188524)
    at async g (/radix-ui-primitives/.yarn/releases/yarn-2.1.1.cjs:2:189885)
    at async q.fetchers.patchPackage (/radix-ui-primitives/.yarn/releases/yarn-2.1.1.cjs:2:204417)
    at async B (/radix-ui-primitives/.yarn/releases/yarn-2.1.1.cjs:2:226253)
    at async Q (/radix-ui-primitives/.yarn/releases/yarn-2.1.1.cjs:2:226716)
    at async v (/radix-ui-primitives/.yarn/releases/yarn-2.1.1.cjs:2:227192)
    at async C.fetchPackageFromCache (/radix-ui-primitives/.yarn/releases/yarn-2.1.1.cjs:2:227431)
    at async q.fetchers.fetch (/radix-ui-primitives/.yarn/releases/yarn-2.1.1.cjs:2:203286)
```

Upgrading Yarn has helped so I didn't dig deeper into this 🤷‍♂️